server/meta.jsx

```diff
@@ -192,6 +192,17 @@ function getSDKScriptAttributes(sdkUrl : ?string, allAttrs : ?{ [string] : strin
     return attrs;
 }
 
+function clearQueryFromLegacySDKUrl(sdkUrl : string ) : string {
```

Comment:
Could we use the name `sanitize` or `transform`?

```suggestion
function sanitizeSDKUrl(sdkUrl : string ) : string {
if ()


```